tox.ini

```diff
@@ -0,0 +1,12 @@
+# Tox (https://testrun.org/tox/latest/) is a tool for running tests
+# in multiple virtualenvs. This configuration file will run the
+# test suite on all supported python versions. To use it, "pip install tox"
+# and then run "tox" from this directory.
+
+[tox]
+envlist = py27, py34, py35, pypy
+
+[testenv]
+commands =
+	{envpython} setup.py install
```

Comment:
Pretty sure this is done automatically.
Not sure if you put it there explicitly for some reason?